dev/release/05-binary-upload.sh

```diff
@@ -128,6 +128,7 @@ docker_run \
   ./runner.sh \
   rake \
     "${rake_tasks[@]}" \
+    ARTIFACTORY_API_KEY="${ARTIFACTORY_API_KEY}" \
     APT_TARGETS=$(IFS=,; echo "${apt_targets[*]}") \
```

Comment:
Oh, thanks! I don't know why I could upload without this...

Could you keep this list in alphabetical order?

```suggestion
    APT_TARGETS=$(IFS=,; echo "${apt_targets[*]}") \
    ARTIFACTORY_API_KEY="${ARTIFACTORY_API_KEY}" \
```